hack/auto-cp.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env sh
+
+set -eu
+# this script prints out a list a commits that are on master that you should probably cherry-pick to the release branch
```

Comment:
this comment is not valid for this script -- looks this was copied from `what-to-cherry-pick.sh`